css/style.css

```diff
@@ -0,0 +1,30 @@
+body{
```

Comment:
Kindly use linters to detect errors in your code.

Reply:
- [ ] Polite comment that clearly states why a change is required 👏🏾